doc/src/minimize.rst

```diff
@@ -89,8 +89,8 @@ be more robust than previous line searches we have tried.  The
 backtracking method is described in Nocedal and Wright's Numerical
 Optimization (Procedure 3.1 on p 41).
 
-The :doc:`minimization styles <min_style>` *quickmin*, *fire* and
-*fire/old* perform damped dynamics using an Euler integration step.  Thus
+The :doc:`minimization styles <min_style>` *quickmin*, and *fire*
```

Comment:
An extra comma snuck in between *quickmin* and *fire*.

Reply:
No accident. This is called an "Oxford comma" (or serial comma): https://en.wikipedia.org/wiki/Serial_comma

Reply:
I am a big proponent of oxford commas, but I was referring to the fact that there are now only 2 styles listed.

Reply:
Right, oxford commas are only used for a list of 3 or more.

Reply:
Ooops. Didn't notice that. Thanks for noticing.